tests/child_thread/enc/enc.cpp

```diff
@@ -32,7 +32,7 @@ int stay_in_ocall_ecall()
 OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
-    true, /* Debug */
-    8,    /* NumHeapPages */
-    8,    /* NumStackPages */
-    4);   /* NumTCS */
+    true, /* AllowDebug */
+    1024, /* HeapPageCount */
+    1024, /* StackPageCount */
+    4);   /* TCSCount */
```

Comment:
Revert the labels in comments. They should match the labels that are recognized in .conf files so shouldn't change unless the .conf file parser is changed to recognize the new names.